docs/plugins/analysis-kuromoji.asciidoc

```diff
@@ -172,6 +172,8 @@ PUT kuromoji_sample
   }
 }
 
+GET _cluster/health?wait_for_status=yellow
```

Comment:
When would we want a doc test to be run *before* the cluster is available? I don't want doc tests to be flaky b/c someone forgot to add this (and it doesn't seem it exists everywhere even now).